apps/designer/app/routes/rest/project/clone.$domain.ts

```diff
@@ -30,7 +30,7 @@ const ensureProject = async ({
  */
 export const loader: LoaderFunction = async ({ request, params }) => {
   if (params.domain === undefined) return { errors: "Domain required" };
-  const user = await authenticator.isAuthenticated(request);
+  const user = await findAuthenticatedUser(request);
```

Comment:
Later same strange logic about `generatedUserId`

Reply:
I guess same, db fails

Reply:
This might be to make this work:

At https://webstudio.is/ you can click "try", and if you're not authenticated, you get authenticated automatically with a new user created.

Reply:
We could actually stop generating users. This was done initially because we didn't have login functionality. We can expect them to signup now with google or github. I think we should remove the auto-generated users logic entirely. Also it is:
1. confusing once you had created a site under a demo user but you never saw the dashboard, so you don't know you can go bak there to actually login and save the sites
A better flow would be to let user signup first, then show them the dashboard, allow them to start from a template (/clone) or a blank project
2. it may be dangerous to keep generating projects without signup when we launch public beta, could be bots or just a wave of users generating too many of them

Reply:
Lets wait though until we get the new design for dashboard with  rewriting this logic, so we can do the new flow completely.